sfdo_template_helpers/fields/markdown.py

```diff
@@ -70,19 +70,34 @@ class MarkdownFieldMixin:
         "a",
     ]
 
-    markdown_attrs = {"img": ["src", "alt", "title"], "a": ["href", "alt", "title"]}
+    allowed_attrs = {"img": ["src", "alt", "title"], "a": ["href", "alt", "title"]}
+
+    property_suffix = "_html"
 
     description = _("Field containing Markdown formatted text.")
 
     def __init__(self, *args, **kwargs):
-        self.property_suffix = kwargs.pop("property_suffix", DEFAULT_SUFFIX)
+        self.property_suffix = kwargs.pop(
+            "property_suffix", MarkdownField.property_suffix
+        )
+        """ an additional property with this suffix is added to the model to render the field, the default value is _html """
+        self.is_safe = kwargs.pop("is_safe", True)
+        """ declare whether or not the html property should be marked safe for the django template system """
+        self.allowed_tags = kwargs.pop("allowed_tags", MarkdownField.allowed_tags)
+        self.allowed_attrs = kwargs.pop("allowed_attrs", MarkdownField.allowed_attrs)
         super().__init__(*args, **kwargs)
 
     def deconstruct(self):
         name, path, args, kwargs = super().deconstruct()
         # Only include kwarg if it's not the default
-        if self.property_suffix != DEFAULT_SUFFIX:
+        if self.property_suffix != MarkdownField.property_suffix:
             kwargs["property_suffix"] = self.property_suffix
+        if not self.is_safe:
+            kwargs["is_safe"] = self.is_safe
+        if self.allowed_tags != MarkdownField.allowed_tags:
+            kwargs["allowed_tags"] = self.allowed_tags
+        if self.allowed_attrs != MarkdownField.allowed_attrs:
```

Comment:
would love some way of making this pattern (and its pair in `__init__`) quicker but meh